src/engines/mod.rs

```diff
@@ -578,7 +580,11 @@ fn input_seek<'a, I: 'a + IoProvider>(es: *mut ExecutionState<'a, I>, handle: *m
         libc::SEEK_SET => SeekFrom::Start(offset as u64),
         libc::SEEK_CUR => SeekFrom::Current(offset as i64),
         libc::SEEK_END => SeekFrom::End(offset as i64),
-        _ => panic!("Unexpected \"whence\" parameter to fseek() wrapper: {}", whence),
+        _ => {
+                tt_warning!(es.status, "Unexpected \"whence\" parameter to fseek() wrapper: {}",
+                            whence);
+                return 0;
```

Comment:
Good point. I have also moved the error conversion from `ExecutionState::input_seek` to `input_seek`. I believe it is better to keep the conversion "Rust error" -> "C error" all in one place in the bridge function.